repose-aggregator/components/filters/header-translation-filter/src/main/scala/org/openrepose/filters/headertranslation/HeaderTranslationFilter.scala

```diff
@@ -65,22 +65,29 @@ class HeaderTranslationFilter @Inject()(configurationService: ConfigurationServi
 
       sourceHeaders foreach { sourceHeader =>
         val originalHeaderName = sourceHeader.getOriginalName
-        val originalHeaderValues = httpRequest.getHeadersScala(originalHeaderName)
+        val originalHeaderValues = if (sourceHeader.isSplittable)
+          httpRequest.getSplittableHeaderScala(originalHeaderName) else
+          httpRequest.getHeadersScala(originalHeaderName)
```

Comment:
Well, the behavior was obvious, but the formatting wasn't. I think it's worth changing, but unless you can find where the Scala style guide describes this situation, you can decide what to change it too. Mostly, if on multiple lines, I would like and expect to see "if" and "else" starting on the same column.